plugins/module_utils/kubevirt.py

```diff
@@ -180,6 +180,8 @@ def _define_datavolumes(self, datavolumes, spec):
             dvt['metadata']['name'] = dv.get('name')
             dvt['spec']['pvc'] = {
                 'accessModes': dv.get('pvc').get('accessModes'),
+                'storageClassName': dv.get('pvc').get('storageClassName'),
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.